core/src/env/srml/srml_only/impls.rs

```diff
@@ -107,6 +112,8 @@ impl<T> Env for SrmlEnv<T>
 where
     T: EnvTypes,
 {
+    type CallError = CallError;
```

Comment:
We do not want to introduce an associated type for `CallError` at this point.

Reply:
Do you have any idea where to implement CallError struct?
I want to use same struct in core/src/env/* and in model/src/exec_env.rs

Reply:
Since `ink_model` depends on `ink_core` just define it in `ink_core` in `ink_core::env` and use it from `ink_model`.